docs/tutorial/Tutorial.md

```diff
@@ -2,30 +2,7 @@
 
 In this tutorial, we will analyze data from the final phase of the [1000 Genomes Project](http://www.internationalgenome.org/about), as described in [A global reference for human genetic variation (Nature 2015)](http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html). We have downsampled the dataset to approximately 10,000 variants consisting of both rare and common variants. We obtained sample information (population, super-population, sex) from the [1000 Genomes website](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20130606_sample_info/20130606_sample_info.xlsx).
 
-Before getting started, read the [Overview](overview.html) section of the docs.
-
-### Install dependencies
-
-In the tutorial, we will run Hail in local mode (using all cores on your laptop, VM, ...) rather than on a Spark cluster. Ensure the following dependencies are installed:
-
-  - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). We recommend the latest version.
-  - [Java 1.8](https://www.google.com/search?q=download+java+8+jdk). Use `java -version` to see which version you currently have.
-  - Python 2.7 and IPython. We recommend the free [Anaconda distribution](https://www.continuum.io/downloads) and will assume this is what's installed below.
-  - [Spark 2](https://spark.apache.org/downloads.html). We recommend the latest version.
-
-If you are setting up a fresh Linux VM with Ubuntu installed (e.g., on GCP or AWS), [this wiki](https://github.com/hail-is/hail/wiki/Install-Hail-dependencies-on-a-fresh-Ubuntu-VM) may help.
-
-### Install Hail
-
-We recommend installation in your home directory. On the command line, download and build Hail by running:
-
-```
-git clone https://github.com/hail-is/hail.git hail
-cd hail
-./gradlew clean shadowJar
-```
-
-This will take a few minutes as additional dependencies are downloaded and Hail is compiled.
+Before proceeding, read the [Overview](pyhail/overview.html) and [Getting_Started](pyhail/getting_started.html) to install and run Hail.
```

Comment:
Linking to the overview is way overkill.  The tutorial can't/shouldn't depend on that background knowledge.

Why the _ in Getting_Started?

"run Hail" sounds strange.